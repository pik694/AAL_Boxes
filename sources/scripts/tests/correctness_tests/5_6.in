5
27 52 66
46 67 24
92 41 2
66 82 52
35 95 96

