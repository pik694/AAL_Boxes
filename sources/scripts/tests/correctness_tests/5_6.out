2 1 3 0 4 
774800->608168
